tests/functional/SqlSyncTest.php

```diff
@@ -25,58 +25,68 @@ public function testSimulatedSqlSync()
         }
 
         $fixtureSites = [
-            'remote' => [
-                'host' => 'server.isp.simulated',
-                'user' => 'www-admin',
-                'ssh' => [
-                    'options' => '-o PasswordAuthentication=whatever'
+                'remote' => [
```

Comment:
Is this really overindented or is that just the diff making it seem so?

Reply:
Should be OK by looking at the code.